### ISSUE TITLE
Issue with building UFS on Cheyenne 

### ISSUE THREAD

**Issue by uturuncoglu** [ufs-community/ufs-weather-model#1281]
## Description
Building UFS model on Cheyenne is failing with error and seems seems could not load the right netcdf module. 

### To Reproduce:
What compilers/machines are you seeing this with?
Give explicit steps to reproduce the behavior.
1. login to Cheyenne
2. check out UFS develop
3. try to run of of the data atmosphere case, `./rt.sh -k -n datm_cdeps_control_cfsr` (not sure others are also failing or not but since issue is loading netcdf library it would be)

## Additional context
NaN

## Output
**output logs**
```
CMake Error at CMakeLists.txt:117 (find_package):
  By not providing "FindNetCDF.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "NetCDF", but
  CMake did not find one.

  Could not find a package configuration file provided by "NetCDF" (requested
  version 4.7.4) with any of the following names:

    NetCDFConfig.cmake
    netcdf-config.cmake

  Add the installation prefix of "NetCDF" to CMAKE_PREFIX_PATH or set
  "NetCDF_DIR" to a directory containing one of the above files.  If "NetCDF"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Comment by jedwards4b** [comment]
FindNetCDF.cmake is provided by PIO in ParallelIO/cmake/FindNetCDF.cmake
it is not part of the netcdf module.

**Comment by uturuncoglu** [comment]
@jedwards4b BTW, this is also failing without updating the CDEPS. So, out-of-box UFS has the issue.

**Comment by DeniseWorthen** [comment]
@uturuncoglu I been running RTs on Cheyenne.intel in the last few days w/o any issue.

**Comment by DeniseWorthen** [comment]
@uturuncoglu Update---I just tried to use compile.sh (outside of rt.sh) and got a different error

/glade/work/worthen/ufs_waveRH/tests/build_fv3/CMakeFiles/CMakeOutput.log

EDIT---I found my (stupid) error. It now seems to be building OK.

**Comment by uturuncoglu** [comment]
@DeniseWorthen strange! let me try again with fresh copy.

**Comment by uturuncoglu** [comment]
@DeniseWorthen I am not sure what was the issue but now it is working also for me. Sorry for false alert! I am closing this issue. Thanks for your help. I have another issue but that could be the CDEPS PR that I am testing with UFS.
